src/rt/util/hash.d

```diff
@@ -1,15 +1,10 @@
 /**
- * This module contains the default hash implementation.
+ * The default hash implementation.
  *
- * Copyright: Copyright Sean Kelly 2009 - 2009.
+ * Copyright: Copyright Sean Kelly 2009 - 2016.
```

Comment:
That would be inaccurate, as the year should be the last year the code was worked on rather than the year the documentation was generated.